backend/src/seed/seed-db.js

```diff
@@ -106,27 +104,27 @@ import { gql } from '../jest/helpers'
     ])
 
     const [racoon, rabbit, wolf, bear, turtle, rhino] = await Promise.all([
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_racoon',
         icon: '/img/badges/indiegogo_en_racoon.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_rabbit',
         icon: '/img/badges/indiegogo_en_rabbit.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_wolf',
         icon: '/img/badges/indiegogo_en_wolf.svg',
       }),
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-03T21:34:48Z" title="Thursday, October 3rd 2019, 11:34:48 pm +02:00">Oct 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -106,27 +104,27 @@ import { gql } from '../jest/helpers'
     ])
 
     const [racoon, rabbit, wolf, bear, turtle, rhino] = await Promise.all([
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_racoon',
         icon: '/img/badges/indiegogo_en_racoon.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_rabbit',
         icon: '/img/badges/indiegogo_en_rabbit.svg',
       }),
-      f.create('Badge', {
+      factory.create('Badge', {
         id: 'indiegogo_en_wolf',
-        icon: '/img/badges/indiegogo_en_wolf.svg',
+        icon: '/img/badges/indiegogo_en_wolfactory.svg',
```


@mattwr18 what is a `wolfactory`? :wolf: :factory: 

:laughing:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-04T08:05:43Z" title="Friday, October 4th 2019, 10:05:43 am +02:00">Oct 4, 2019</time>_

---


haahahah global find and replace :rofl: